File: reset-assured/src/test/java/tests/UI/GitHubUITests.java
package tests.UI;

import Base.UI.BaseTest;
import GitHub.API.IssuesApi;
import GitHub.Objects.IssueDetails;
import GitHub.UI.IssuesPage;
import org.testng.annotations.Test;

import java.util.HashMap;
import java.util.List;

import static org.testng.Assert.assertEquals;
import static org.testng.AssertJUnit.assertTrue;

public class GitHubUITests extends BaseTest {
    @Test
    public void verifyOpenAreLessThanClosed() {
        IssuesPage issues = new IssuesPage(driver);
        int open = issues.getOpenIssuesCount();
        int closed = issues.getClosedIssuesCount();
        System.out.println("Open: " + String.valueOf(open));
        System.out.println("Closed: " + String.valueOf(closed));
        assertTrue("Open issues are more than closed.", open < closed);
    }

    @Test
    public void verifyIssuesCountIsCorrect() {

        // Get open and closed count via UI
        IssuesPage issues = new IssuesPage(driver);
        int openUI = issues.getOpenIssuesCount();

        // Get open and closed count via API
        IssuesApi issuesApi = new IssuesApi("dtopuzov", "test");
        HashMap<String, String> openFilter = new HashMap<>();
        openFilter.put("state", "open");
        int openAPI = issuesApi.getIssues(openFilter).size();
        assertEquals(openUI, openAPI, "Issues count is wrong!");
    }
}

File: 2018/github-tests-nunit/GitHubTests/Tests/UI/GitHubUITests.cs
using GitHubTests.Base;
using GitHubTests.GitHub.API;
using GitHubTests.GitHub.UI;
using NUnit.Framework;
using System.Configuration;

namespace GitHubTests
{
    [TestFixture]
    public class GitHubUITests : WebTest
    {
        private readonly static string User = ConfigurationManager.AppSettings["User"];
        private readonly static string Repo = ConfigurationManager.AppSettings["Repo"];

        private IssuesPage issuesPage;

        [SetUp]
        public void LoginTestInit() => issuesPage = new IssuesPage(Driver);

        [Test]
        public void OpenIssuesAreLessThenClosed()
        {
            int open = issuesPage.GetOpenIssuesCount();
            int closed = issuesPage.GetClosedIssuesCount();
            Assert.IsTrue(open < closed, "Open issues are more than opened!");
        }

        [Test]
        public void OpenIssuesCountIsCorrect()
        {
            IssuesAPI issuesAPI = new IssuesAPI();
            int openUI = issuesPage.GetOpenIssuesCount();
            int openAPI = issuesAPI.GetAllIssues(organization: User, repository: Repo, options: "state=open").Count;
            Assert.AreEqual(openUI, openAPI, "API and UI report different issues count!");
        }
    }
}
